next/components/forms/FieldGroups.tsx

```diff
@@ -0,0 +1,170 @@
+import { UploadMinioFile } from '@backend/dtos/minio/upload-minio-file.dto'
```

Comment:
please split these into multiple files - perhaps we can create a subfolder, so it will look like `compontents/forms/groups/DateFromTo.tsx`, `compontents/forms/groups/TimeFromTo.tsx` etc.